variables.tf

```diff
@@ -172,6 +172,13 @@ variable "identity_type" {
   }
 }
 
+variable "idle_timeout_in_minutes" {
+  type        = number
+  description = "(Optional) Desired outbound flow idle timeout in minutes for the cluster load balancer. Must be between 4 and 120 inclusive."
```

Comment:
Could we backquote the number as [official document](https://github.com/hashicorp/terraform-provider-azurerm/blob/f8e47e77eaa8baf00918d6172edefe97fdfaaa7b/website/docs/r/kubernetes_cluster.html.markdown?plain=1#L567) did here? And I think we should declare the default value in the description too.